### DISCUSSION TITLE
Incorrect Value My UPS

### DISCUSSION THREAD

**Discussion by jameshlog** [nutdotnet/WinNUT-Client#155] [Q&A]
UPS Model : APC Smart-UPS SMC 1000IC (02.0 (1061))
NAS : Synology DS220+ (DSM 7.2.1-69057 Update 5)
Windows Version : Windows 10 Pro (22H2)
Winnut Version : 2.3.8892.21649

Hi,

with the windows client 'Input Voltage, Output Voltage, UPS Load, Battery Voltage' indicator not working.
I think this value is wrong. currently using the Smart Connect trial version. but, remaing time value is correct. 

here my smart connect website UPS Value and Winnut Value

![image](https://github.com/nutdotnet/WinNUT-Client/assets/116254061/b0091783-ae34-487d-9d2d-2249358a9531)

![image](https://github.com/nutdotnet/WinNUT-Client/assets/116254061/a059c607-fd31-42bb-8720-99d90cea1912)

check the correct value in Winnut?

**Comment by gbakeman** [comment]
Hi @jameshlog ,

I want to link back to your [previous issue](https://github.com/gawindx/WinNUT-Client/issues/189) and pull in the variables you posted:

<details>
<summary>UPS Variables</summary>
```
ups (American Power Conversion/Smart-UPS_1000/UPS 02.0 / ID=1061)
battery.charge (Description unavailable) : 100
battery.charge.low (Description unavailable) : 10
battery.charge.warning (Description unavailable) : 50
battery.runtime (Description unavailable) : 1085
battery.runtime.low (Description unavailable) : 150
battery.type (Description unavailable) : PbAc
battery.voltage (Description unavailable) : 26.3
battery.voltage.nominal (Description unavailable) : 24.0
device.mfr (Description unavailable) : American Power Conversion
device.model (Description unavailable) : Smart-UPS_1000
device.serial (Description unavailable) : * Encryption *
device.type (Description unavailable) : ups
driver.name (Description unavailable) : usbhid-ups
driver.parameter.pollfreq (Description unavailable) : 30
driver.parameter.pollinterval (Description unavailable) : 5
driver.parameter.port (Description unavailable) : auto
driver.parameter.synchronous (Description unavailable) : no
driver.version (Description unavailable) : DSM7-2-1-NewModel-repack-64570-230831
driver.version.data (Description unavailable) : APC HID 0.96
driver.version.internal (Description unavailable) : 0.41
ups.beeper.status (Description unavailable) : enabled
ups.delay.shutdown (Description unavailable) : 20
ups.firmware (Description unavailable) : UPS 02.0 / ID=1061
ups.mfr (Description unavailable) : American Power Conversion
ups.mfr.date (Description unavailable) : 2023/10/19
ups.model (Description unavailable) : Smart-UPS_1000
ups.productid (Description unavailable) : 0003
ups.serial (Description unavailable) : ****
ups.status (Description unavailable) : OL
ups.timer.reboot (Description unavailable) : -1
ups.timer.shutdown (Description unavailable) : -1
ups.vendorid (Description unavailable) : 051d
```
</details>

I'd have to reiterate one thing I said which is that your NUT server is not providing WinNUT with any useful variables other than battery information. Why that is isn't very clear to me, especially since APC's software seems to be reading it directly just fine. So we know that the UPS can read that information just fine. In addition, [here's](https://networkupstools.org/ddl/APC/Smart-UPS_1000.html) a hardware compatibility page for your UPS. It seems like the NUT server should be reporting much more than it currently is. I think if you want to get this working correctly, you'll need to troubleshoot your NUT server.

Something I've noticed is that you're using the Synology distribution of NUT server software, which may be outdated. Could you find a way to [download the latest version](https://networkupstools.org/download.html) of NUT server software, and try connecting your UPS to that?

**Reply by jameshlog** [reply]
Thank you for your answer. I tried install other PC latest NUT server software.
However, there is still no change in the setting value. 
The input voltage, frequency, output voltage, and battery voltage values appear to be normal, but there is no way to check them.
Remaining Time, Battery Charge value is appears to be the correct value.
Both values are somewhat consistent with those confirmed by Synology.
I don't know if this problem is with the APC UPS or my Synology NAS. Other users did not encounter this problem.
Is there another method I can try? I would like to help with this project. For anyone going through similar problems as me.

**Reply by gbakeman** [reply]
I'm afraid I'm getting out of my depth troubleshooting the NUT server. You may want to [reach out to the NUT project](https://networkupstools.org/support.html) for further support. Specifically, you can post on the [User Mailing List](http://lists.alioth.debian.org/mailman/listinfo/nut-upsuser), [post a new issue](https://github.com/networkupstools/nut/issues) in the NUT GitHub, or even seek help in their IRC channel. Either way, make sure you provide as much information as possible. Specifically, they want:

- OS name and version,
- exact NUT version,
- NUT installation method: package, or a custom build from source tarball or GitHub (which fork, branch, PR),
- exact device name and related information (manufacturing date, web pointers, …),
- complete problem description, with any relevant traces, like system log excerpts, and driver debug output. You can obtain the latter using the following command, running as root and after having stopped NUT: `/path/to/driver -DD -a <upsname>`

I also came across [another user seeking help](https://alioth-lists.debian.net/pipermail/nut-upsuser/2015-February/009567.html) with your model of UPS, and the information that was requested is:

- `lsusb -vvv`
- `cat /etc/nut/ups.conf`
- `/lib/nut/usbhid-ups -u root -a <your ups name>`

I'm sorry I can't help much going further, but if you do end up posting for help there, please cross link your post here so other people who find this can find your support post. And please also post back here if you've made any progress. :)

**Comment by itgmkol** [comment]
I have the same problem on qnap. Do you have a solution?

**Reply by gbakeman** [reply]
Hi,

Please take a look through @jameshlog's issue report [here](https://github.com/gawindx/WinNUT-Client/issues/189), and see if any of it helps. Otherwise, please make a reply here and supply the information that was requested of them and I'll see if I can give you some answers (may be similar to what I commented above.)
